spec/components/schemas/content/optin_request.ts

```diff
@@ -0,0 +1,62 @@
+import { SchemaObject } from 'openapi3-ts';
+import { ref as baseRef } from './base';
+import { createComponentRef } from '../../../../utils/ref';
+
+const notificationOptIn: SchemaObject = {
+  type: 'object',
+  allOf: [{
+    $ref: baseRef,
+  }, {
+    type: 'object',
+    properties: {
+      type: {
+        type: 'string',
+        example: 'optin_request',
+      },
+      medialUrl: {
+        description: 'URL for the thumbnail the user will see.',
+        type: 'string',
+        example: 'https://cdn.zenvia.com/main-picture.jpg',
+      },
+      title: {
+        description: 'The topic of the recurring notification subscription.',
```

Comment:
Como no response acabei sugerindo evitar o notification, especialmente ao ver que no texto da doc do Facebook nem sempre usam notifcation, e message é uma conceito melhor pra nossa API, que tal:
```
The topic of the recurring messages subscription. This will be displayed in the opt-in request.
```